x-pack/plugins/uptime/public/uptime_app.tsx

```diff
@@ -126,161 +128,176 @@ class Application extends React.Component<UptimeAppProps, UptimeAppState> {
-          <EuiPage className="app-wrapper-panel">
-            <EuiHeader>
-              {/*
+      <I18nProvider>
+        <Router basename={routerBasename}>
+          <ApolloProvider client={graphQLClient}>
+            <EuiPage className="app-wrapper-panel">
+              <EuiHeader>
+                {/*
               // @ts-ignore TODO no typings for grow prop */}
-              <EuiHeaderSection grow={true}>
-                <EuiHeaderSectionItem border="right">
-                  <EuiHeaderLogo
-                    aria-label="Go to Uptime home page"
-                    href="#/"
-                    iconType="heartbeatApp"
-                    iconTitle="Uptime"
-                  >
-                    Uptime
-                    <EuiIcon style={{ paddingLeft: '8px' }} size="xl" type="beaker" />
-                  </EuiHeaderLogo>
-                </EuiHeaderSectionItem>
-                {!isUsingK7Design && (
-                  <EuiHeaderSectionItem>
-                    <div style={{ paddingTop: '20px', paddingRight: '8px' }}>
-                      <EuiHeaderBreadcrumbs breadcrumbs={this.state.breadcrumbs} />
+                <EuiHeaderSection grow={true}>
+                  <EuiHeaderSectionItem border="right">
+                    <EuiHeaderLogo
+                      aria-label={i18n.translate('xpack.uptime.appHeader.icon.ariaLabel', {
+                        defaultMessage: 'Go to Uptime home page',
+                      })}
+                      href="#/"
+                      iconType="heartbeatApp"
+                      iconTitle={i18n.translate('xpack.uptime.appHeader.icon.title', {
+                        defaultMessage: 'Uptime',
+                      })}
+                    >
+                      <FormattedMessage
+                        id="xpack.uptime.appHeader.logo.text"
+                        defaultMessage="Uptime"
+                      />
+                      <EuiIcon style={{ paddingLeft: '8px' }} size="xl" type="beaker" />
+                    </EuiHeaderLogo>
+                  </EuiHeaderSectionItem>
+                  {!isUsingK7Design && (
+                    <EuiHeaderSectionItem>
+                      <div style={{ paddingTop: '20px', paddingRight: '8px' }}>
+                        <EuiHeaderBreadcrumbs breadcrumbs={this.state.breadcrumbs} />
+                      </div>
+                    </EuiHeaderSectionItem>
+                  )}
+                </EuiHeaderSection>
+                <EuiHeaderSection side="right">
+                  <EuiHeaderSectionItem border="none">
+                    <div style={{ marginTop: '10px', marginLeft: '8px' }}>
+                      <EuiDatePickerRange
+                        startDateControl={
+                          <EuiDatePicker
+                            selected={moment(this.state.dateRangeStart)}
+                            isInvalid={dateRangeIsInvalid()}
+                            aria-label="Start Date"
```

Comment:
I think this label should be translated as well.

Reply:
Resolved in https://github.com/elastic/kibana/pull/28183/commits/689a403b127f28a2ba13fcbc4cf570bcbb1eb0bc.